opentreemap/treemap/views/misc.py

```diff
@@ -227,3 +229,20 @@ def instance_geojson(instance):
                  }))
 
     return [instance_geojson(instance) for instance in instances]
+
+
+def error_page(template, status_code):
+    def inner_fn(request):
+        # API requests with an unhandled error should return JSON, not HTML
+        if request.path.startswith('/api/'):
```

Comment:
I think we should _always_ return JSON for API requests.  That said, returning JSON errors for _other_ requests that are expecting a JSON response is almost certainly a good idea, will do.